libraries/AP_Mission/AP_Mission.h

```diff
@@ -203,6 +203,7 @@ class AP_Mission
         bool start_control; // start or stop engine
         bool cold_start; // use cold start procedure
         uint16_t height_delay_cm; // height delay for start
+        bool allow_disarmed_start; // allow starting the engine while disarmed
```

Comment:
Lets just consider moving this straight to a 16 bit flag, rather then keeping this minimal bool.